reader/processor/processor.go

```diff
@@ -49,6 +49,39 @@ func ProcessFeedEntries(store *storage.Storage, feed *model.Feed) {
 		// The sanitizer should always run at the end of the process to make sure unsafe HTML is filtered.
 		entry.Content = sanitizer.Sanitize(entry.URL, entry.Content)
 	}
+
+	filterFeedEntries(feed)
+}
+
+/*
+Filters feed entries based on regex rules
+First we filter based on our keep list, then we remove those entries that match the block list
+*/
+func filterFeedEntries(feed *model.Feed) {
```

Comment:
Do you mind adding some unit tests for this function? Just to make sure there is no regression in the future.

Reply:
@fguillot done.